mlops/kubeflow/robustness_checker_pytorch/src/model_files/__init__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 IBM Corporation 
+# 
+# Licensed under the Apache License, Version 2.0 (the "License"); 
+# you may not use this file except in compliance with the License. 
+# You may obtain a copy of the License at 
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0 
+# 
+# Unless required by applicable law or agreed to in writing, software 
+# distributed under the License is distributed on an "AS IS" BASIS, 
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+# See the License for the specific language governing permissions and 
+# limitations under the License. 
+
```

Comment:
What is this file doing?

Reply:
For PyTorch models, we need the users to also provide the model class file and we will load that class as python module. For python, it's often recommended/required to have a `__init__.py` inside the python module directory even though this is an empty file.

For more details you can refer to https://docs.python.org/3/tutorial/modules.html#packages and https://pythontips.com/2013/07/28/what-is-__init__-py/